service/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/FaReportMessageFishingTrips.java

```diff
@@ -0,0 +1,26 @@
+package eu.europa.ec.fisheries.uvms.activity.service;
+
+import eu.europa.ec.fisheries.uvms.activity.fa.entities.FishingActivityEntity;
+import eu.europa.ec.fisheries.uvms.activity.fa.entities.FishingTripEntity;
+import eu.europa.ec.fisheries.uvms.activity.fa.entities.FishingTripKey;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FaReportMessageFishingTrips {
```

Comment:
Naming - so far it only handles trips and activities, right? Maybe name should end with cache or something.